ledger/heavy/executor/jetkeeper.go

```diff
@@ -284,15 +355,18 @@ func infoToList(s map[insolar.JetID]struct{}) []insolar.JetID {
 	return r
 }
 
-func (jk *dbJetKeeper) checkPulseConsistency(ctx context.Context, pulse insolar.PulseNumber) bool {
+func (jk *dbJetKeeper) checkPulseConsistency(ctx context.Context, pulse insolar.PulseNumber, checkBackup bool) bool {
 	expectedJets := jk.jetTrees.All(ctx, pulse)
```

Comment:
This is a fundomental problem. We should calculate expected jets only from finilized pulses.

Reply:
done